.evergreen/.evg.yml

```diff
@@ -550,9 +600,6 @@ functions:
 
 # Anchors
 
-hosts: &hosts
```

Comment:
You may have a rebasing issue, since this looks like a change that I recently made to fix an Evergreen issue.

Reply:
I have rebased with the latest commits in `master`.